contrib/scripts/kind.sh

```diff
@@ -0,0 +1,107 @@
+#!/usr/bin/env bash
+
+# Source: https://kind.sigs.k8s.io/docs/user/local-registry/
+
+set -euo pipefail
+
+default_controlplanes=1
+default_workers=1
+default_cluster_name=""
+default_image=""
+
+PROG=${0}
+CONTROLPLANES="${1:-${default_controlplanes}}"
+WORKERS="${2:-${default_workers}}"
+CLUSTER_NAME="${3:-${default_cluster_name}}"
+# IMAGE controls the K8s version as well (e.g. kindest/node:v1.11.10)
+IMAGE="${4:-${default_image}}"
+
+usage() {
+  echo "Usage: ${PROG} [control-plane node count] [worker node count] [cluster-name] [node image]"
+}
+
+have_kind() {
+    [[ -n "$(command -v kind)" ]]
+}
+
+if ! have_kind; then
+    echo "Please install kind first:"
+    echo "  https://kind.sigs.k8s.io/docs/user/quick-start/#installation"
+fi
+
+if [[ "${#}" -gt 4 ]]; then
+  usage
+  exit 1
+fi
+
+if [[ "${#}" -gt 4 ]] ||
+   [[ "${CONTROLPLANES}" == "-h" ||
+      "${CONTROLPLANES}" == "--help" ]]; then
+  usage
+  exit 0
+fi
+
+# Registry will be localhost:5000
+reg_name="kind-registry"
+reg_port="5000"
+running="$(docker inspect -f '{{.State.Running}}' "${reg_name}" 2>/dev/null || true)"
+if [[ "${running}" != "true" ]]; then
+  docker run \
+    -d --restart=always -p "${reg_port}:5000" --name "${reg_name}" \
+    registry:2
+fi
+
+kind_cmd="kind create cluster"
+
+if [[ -n "${CLUSTER_NAME}" ]]; then
+  kind_cmd+=" --name ${CLUSTER_NAME}"
+fi
+if [[ -n "${IMAGE}" ]]; then
+  kind_cmd+=" --image ${IMAGE}"
+fi
+
```

Comment:
Might also need to set context - `kubectl cluster-info --context kind-kind`

Reply:
I think the `kind` command when it creates the cluster, it automatically sets the context

Reply:
Yeah, that has been my observation lately as well.